src/source/Metrics/Metrics.c

```diff
@@ -125,10 +151,18 @@ STATUS rtcPeerConnectionGetMetrics(PRtcPeerConnection pRtcPeerConnection, PRtcRt
             getIceCandidatePairStats(pRtcPeerConnection, &pRtcMetrics->rtcStatsObject.iceCandidatePairStats);
             break;
         case RTC_STATS_TYPE_LOCAL_CANDIDATE:
-            getIceCandidateStats(pRtcPeerConnection, &pRtcMetrics->rtcStatsObject.localIceCandidateStats);
+            CHK_STATUS(getIceCandidateStats(pRtcPeerConnection, &pRtcMetrics->rtcStatsObject.localIceCandidateStats, FALSE));
+            if (NULL != getenv(LOG_STATS)) {
```

Comment:
I would recommend against using getenv this deep in the stack. Normally, what we do is we use getenv in the samples and set appropriate flags in the objects which get used in the deeper layers later.

Reply:
I have removed internal logging completely. Going to keep it consistent with the way signaling stats is written and allow application to take care of logging.